resources/js/main.js

```diff
@@ -94,26 +97,43 @@ function createAButton(buttonName, className, buttonSVG) {
   buttonName.innerHTML = buttonSVG;
   return buttonName;
 }
-// Adds a new item to the todo list
-function addItemToDOM(text, completed) {
-  var list = (completed) ? document.getElementById('completed') : document.getElementById('todo');
 
+function createLiWithButtons(text) {
   var item = document.createElement('li');
   item.innerText = text;
   var buttonElements = document.createElement('div');
   buttonElements.classList.add('buttons');
+  return {
+    item,
+    buttonElements
+  }
+}
+
+function addEventListenerToButton() {
```

Comment:
this method have too much responsibilities. It attached 2 click events and also create two buttons. This functionality should be separate.

Reply:
I mean this method need to be refactored `addEventListenerToButton`